appengine-java11/quarkus-helloworld/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: java11
+instance_class: F1
+# The entrypoint here is mandatory as the appengine-staging area contains an
+# exploded fatjar.
+entrypoint: 'java -XX:MaxRAM=128m -XX:ActiveProcessorCount=2 -Xmx32m -Dquarkus.log.file.enable=false -Dquarkus.http.host=0.0.0.0 -jar quarkus-helloworld-1.0-SNAPSHOT-runner.jar'
```

Comment:
Do you get two processors in GAE?  I've recently heard yes, but do you know that is true?